sorc/link_fv3gfs.sh

```diff
@@ -287,10 +284,10 @@ cd ${pwd}/../sorc   ||   exit 8
     $SLINK gsi.fd/util/Radiance_Monitor/nwprod/radmon_shared.v3.0.0/sorc/verf_radbcoef.fd  radmon_bcoef.fd
 
     [[ -d radmon_bcor.fd ]] && rm -rf radmon_bcor.fd
-    $SLINK gsi.fd/util/Radiance_Monitor/nwprod/radmon_shared.v3.0.0/sorc/verf_radbcor.fd   radmon_bcor.fd 
+    $SLINK gsi.fd/util/Radiance_Monitor/nwprod/radmon_shared.v3.0.0/sorc/verf_radbcor.fd   radmon_bcor.fd
```

Comment:
Does this version number need to go too? Also line 290.

Reply:
It would have to go.  There is no versioning in the GSI repository. Stay tuned.

Reply:
Yes, the version numbers lines 278, 281, 284, 287, 290, and 293 will all need to be removed as well.

Reply:
done.